### ISSUE TITLE
Custom package and path for bind data file

### ISSUE THREAD

**Issue by komyg** [asticode/go-astilectron-bundler#33]
Hi,

I am using the bundler to generate a bind-data file for my project. Since I am using a third party build tool, I decided to go with only the `bd` option (the full command is:  `astilectron-bundler bd`). 

However I noticed that the bind-data file is always generated in the project root and in is on the `main` package. Is it possible to choose to which package this file must belong to?

Thanks,
Felipe

**Comment by komyg** [comment]
Hi,

Yes, but it is not working. My config file is below:

```
{
  "icon_path_windows": "./public/logo-onyo.ico",
  "resources_path": "./public",
  "bind": {
    "output_path": "./data",
    "package": "data"
  }
}
```

However the bind-data file is still being generated in the root folder using the main package.

**Comment by asticode** [comment]
Did you remember to `go install` the bundler so that binary is up to date?

**Comment by komyg** [comment]
I ran the commands below:

```
$ go get -u github.com/asticode/go-astilectron-bundler/...
$ go install github.com/asticode/go-astilectron-bundler/astilectron-bundler
```

But the problem remains.

**Comment by komyg** [comment]
I found out what the problem was: for some reason I had two GOPATHs configured on my computer: one was the one I created when I installed Go and the other was inside my user profile: [User]\Go. 

I believe that the second variable was created by the Go installer on its own when it upgraded to the version 1.11.

Anyway I removed the duplicated variable and the folder it pointed to and everything works.

Thanks for all the help! :)
